packages/util-package-renderer/lib/js/js-helper.js

```diff
@@ -42,10 +59,13 @@ const transpileJS = async (packageRoot, demoCodeFolder) => {
 		throw error;
 	}
 
-	// Output as ES module file, for inclusion in a <script type="module"> tag
+	// Output as iife for the browser
```

Comment:
why?

Reply:
browser support. Do we want the living styleguide to work in all browsers?

Reply:
looks like i wasn't too bothered about that when i wrote this 😂  but sure :)